app/controllers/AdminForms.scala

```diff
@@ -229,12 +229,13 @@ object AdminForms {
     )(ParamData.apply)(ParamData.unapply)
   )
 
-  case class ThreadPoolData(threadPoolKey: String, coreSize: Int)
+  case class ThreadPoolData(threadPoolKey: String, coreSize: Int, queueSize: Int)
 
   val threadPoolForm = Form(
     mapping(
       "threadPoolKey" -> text,
-      "coreSize" -> number
+      "coreSize" -> number(min = 1),
+      "queueSize" -> number(min = 0)
```

Comment:
That isn't the default, that is a constraint on the min value, which is 0 for no queue (must immediately execute)